### ISSUE TITLE
MaterialListValueBox with Empty Placeholder setSelectedValue throws NullPointerException

### ISSUE THREAD

**Issue by Fuzzo** [GwtMaterialDesign/gwt-material#784]
In the class MaterialListValueBox there is the following method:
```java 
   public int getIndex(T value) {
        int count = getItemCount();
        for (int i = 0; i < count; i++) {
            if (getValueInternal(i).equals(value)) {
                return i;
            }
        }
        return -1;
    }
```
that is bugged in case the MaterialListValueBox has an empty placeholder.
In this case, getValueInternal(i) returns null that throw a NullPointerException.

**Comment by kevzlou7979** [comment]
What version of GMD are you using ?

**Comment by Fuzzo** [comment]
I'm using the v2.0.1

**Comment by kevzlou7979** [comment]
Ok will take a look on this.

**Comment by kevzlou7979** [comment]
Just check the commit history  and found out that we've got this fixed on 2.1 https://github.com/GwtMaterialDesign/gwt-material/commit/45974d6f0d9be24436d379f020662fde292979c8
